src/extension.ts

```diff
@@ -317,6 +321,35 @@ function recordToolkitInitialization(activationStartedOn: number, logger?: Logge
     }
 }
 
+/**
+ * Wraps the `vscode.window.withProgress` functionality with functionality that also writes to the output channel.
+ *
+ * Cloud9 does not show a progress notification.
+ */
+function wrapWithProgressForCloud9(channel: vscode.OutputChannel): typeof vscode.window['withProgress'] {
```

Comment:
should this live in shared/vscode/ or somewhere like that?

Reply:
Depends on how visible we want this behavior/edge-case. Previously it was kind of tucked away so putting it in the extension entry point makes it more visible, maybe too much?